timesketch/lib/datastores/elastic.py

```diff
@@ -415,6 +412,65 @@ def search_stream(self, sketch_id=None, query_string=None,
             for event in result['hits']['hits']:
                 yield event
 
+    # pylint: disable=line-too-long
```

Comment:
why this pylint disable?

Reply:
Some line overflow in the filter dict, but to remain readably and consistent I opted to ignore this lint error. I moved it closer to where it ignores it though.